packages/create-sitecore-jss/src/templates/nextjs/src/pages/[[...path]].tsx

```diff
@@ -6,6 +6,7 @@ import { GetServerSideProps } from 'next';
 <% } -%>
 import NotFound from 'src/NotFound';
 import Layout from 'src/Layout';
+import SafeHydrate from './../components/SafeHydrate';
```

Comment:
One last thought... Should we move the `SafeHydrate` component up to the src root (next to `Layout`)? @illiakovalenko what do you think?

Reply:
@ambrauer Agree with you, we don't need to put this component under `components`, because it will be added to `componentFactory` in this case